compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -554,12 +554,21 @@ error.invalid.action.invocation=\
 error.undefined.action=\
   undefined remote function ''{0}'' in client object {1}
 
-error.tainted.value.passed.to.sensitive.parameter=\
-  tainted value passed to sensitive parameter ''{0}''
+error.tainted.value.passed.to.untainted.parameter=\
+  tainted value passed to untainted parameter ''{0}''
+
+error.tainted.value.passed.to.untainted.param.in.obj.method=\
+  tainted value ''{0}'' passed to untainted parameter originating from object method invocation
 
 error.tainted.value.passed.to.global.variable=\
   tainted value passed to global variable ''{0}''
 
+error.tainted.value.passed.to.global.object=\
+  tainted value passed to global object ''{0}''
```

Comment:
Do we need says which tainted value? 

How about rewording, this to `tainted value ''{0}'' passed to module object variable ''{0}''`

Reply:
As tainted value could be coming from any expression, we can't say which tainted value. I'll change the other part.